Sources/PointFree/Episode/Show.swift

```diff
@@ -210,39 +270,41 @@ func userEpisodePermission<I, Z>(
       .map { conn.map(const($0 .*. conn.data)) }
 }
 
-
-private let episodeNotFoundView = { param, user, subscriberState, route in
+private func episodeNotFoundView(
```

Comment:
the `episodeNotFoundView` was previously super super messy, taking a whole bunch of params that weren't necessary and then were just ignored with `_`s. I finally cleaned it up a bit, though I think the ergonomics can be improved a lot more.